feature_apps/src/main/java/cm/aptoide/pt/feature_apps/data/App.kt

```diff
@@ -98,3 +99,65 @@ val emptyApp = App(
   obb = null,
   developerName = ""
 )
+
+val mockApp = App(
```

Comment:
What is the difference with `emptyApp`?
Mb better to replace both of these with a random generator? 😄

Reply:
Delivered without PR